app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    source_file, destination_path = parts[1], parts[2]
+
+    if os.path.exists(source_file):
+        if destination_path.endswith("/"):
+            if not os.path.exists(destination_path):
+                os.makedirs(destination_path)
```

Comment:
You could avoid using this if condition if you pass `exist_ok=True` flag to the `makedirs`